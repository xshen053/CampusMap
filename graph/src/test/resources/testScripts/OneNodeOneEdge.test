# Create a graph
CreateGraph ONOE_graph

# Add a node
AddNode ONOE_graph A

# Add an edge
AddEdge ONOE_graph A A e1

# Print the nodes in the graph and the outgoing edges from A
ListNodes ONOE_graph
ListChildren ONOE_graph A