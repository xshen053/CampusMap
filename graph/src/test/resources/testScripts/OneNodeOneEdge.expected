# Create a graph
created graph ONOE_graph

# Add a node
added node A to ONOE_graph

# Add an edge
added edge e1 from A to A in ONOE_graph

# Print the nodes in the graph and the outgoing edges from A
ONOE_graph contains: A
the children of A in ONOE_graph are: A(e1)